File: Domain/Patient.cs
using System;

namespace Domain
{
    public class Patient
    {

        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string Gender { get; set; }
        public long HealthCardNumber { get; set; }
        public string VersionCode { get; set; }
    }

}

File: frontend/src/app/models/patient.ts


export interface IPatientEnvelope{
    patients: IPatient[];
    patientsCount: number;

}

export interface IPatientFormValues{
    firstName: string;
    lastName : string;
    dateOfBirth: Date;
    gender: string;
    healthCardNumber: number;
    versionCode: string;
}

export interface IPatient extends IPatientFormValues{
    id: number
}